src/main/resources/application.yaml

```
@@ -0,0 +1 @@
+logging.level.org.zalando.logbook: WARN
```

Данный синтаксис соответствует структуре `properties`.
`yaml` использует отступы и структуру вложенности, что делает его более читаемым:
```suggestion
logging:
  level:
    org:
      zalando:
        logbook: WARN
```

---

исправлено